src/locales/nl/parsers/NLTimeUnitCasualRelativeFormatParser.ts

```diff
@@ -4,18 +4,21 @@ import { ParsingComponents } from "../../../results";
 import { AbstractParserWithWordBoundaryChecking } from "../../../common/parsers/AbstractParserWithWordBoundary";
 import { reverseTimeUnits } from "../../../utils/timeunits";
 
-const PATTERN = new RegExp(`(deze|vorige|afgelopen|komende|over|\\+|-)\\s*(${TIME_UNITS_PATTERN})(?=\\W|$)`, "i");
+const PATTERN = new RegExp(`(dit|deze|vorig|afgelopen|(aan)?komend|over|\\+|-)e?\\s*(${TIME_UNITS_PATTERN})(?=\\W|$)`, "i");
```

Comment:
Same here. It is better to use a non-capturing group if you don't really intend to capture the "aan" part.

Reply:
Sure. Done.